modules/intro/pages/applications.adoc

```diff
@@ -40,10 +40,107 @@ Duplicating an application is useful for quickly trying out new options while st
 
 === Share an application
 
-Click on btn:[Share] in the three-dot menu to bring up the Share window.
+==== Share via login
+* Click on btn:[Share] in the three-dot menu to bring up the Share window.
+
+* Select one of the listed users.
 
 image::share-application.png[]
 
+==== Share via access token
+
+If you want to share your widget with a user who does not have an Insights login, you can send them an access token.
+
+==== Creating database account
+
+You will need a separate database account to generate an access token for Insights.
+Ensure that you grant minimal privileges to this account.
+
+Please check the xref:tigergraph-server:reference:list-of-privileges.adoc[].
+
+TigerGraph provides two modes for running queries:
+
+* Run a previously **Installed** query
+* **Interpret** an ad hoc query
+
+For Insights, **Interpret** query is the default query mode, unless you explicitly convert it to an **Installed** query.
+If your widget only needs pre-designed queries, then you may only need **installed** queries.
+If you want users to be able to create and run their own queries, then likely they will need to be able to **interpret** queries.
+
+==== Query permissions required to write/run queries.
+
+`WRITE_QUERY` privilege is required to interpret queries or to install queries.
+It is not needed to run an installed query.
+
+`READ_DATA` privilege is required for any type of query, and you can grant it at various levels.
+(Details are provided in xref:gui:admin-portal:management/user-management.adoc[local privileges])
+
+`READ_SCHEMA` and `READ_QUERY` are also needed for any type of query.
+
+.The minimal required privileges for different types of queries:
+[cols='2']
+|===
+| Query Type | Minimal Required Privileges
+
+| Installed Query | `READ_DATA` ,`READ_SCHEMA`, `READ_QUERY`
+| Interpret Query | `READ_DATA` ,`READ_SCHEMA` ,`READ_QUERY`, `WRITE_QUERY`
+|===
+
+==== (Optional) Config nginx
+
+This is only needed if you want to embed Insights into third party webpages.
+
+* Switch to the Admin Portal tool to configure `nginx`.
+* Set the `content-security-policy` parameter as below, substituting in your web page's URL.
+
+[source, console]
+Content-Security-Policy: frame-ancestors 'self' https://www.thirdparty.com;
+
+==== Generating access token
+
+Use the account you just created to generate an access token.
+The access token is inside a URL that can be embedded on a page as an iframe into a dashboard.
+
+* Go to `Application > Settings` in the three-dot menu of the specific application.
```

Comment:
FYI, I (re)discovered this weekend that Antora has special syntax for buttons, menus, and keyboard keys. 
https://docs.antora.org/antora/latest/asciidoc/ui-macros/
Instead of `Application > Settings`, you could use
menu:Application[ Settings ].

Reply:
great tip! I utilized it in latest commit.